.travis.yml

```diff
@@ -37,6 +37,11 @@ jobs:
         - travis_wait rustup component add rustfmt-preview
         - ./autogen.sh && ./configure --without-makeinfo --with-x=no --with-ns=no --without-gconf --without-gsettings
       script: ./.travis-format.sh
```

Comment:
Why are you running it twice? To catch mistakes in the code under different `cfg` options? if so, the comment should go and this could be restructured like the unit test runs further down.

Reply:
There's an error in the mac logs `error: 'rustfmt' is not installed for the toolchain 'nightly-2019-05-01-x86_64-apple-darwin'`. I want to get the mac tests green.

Reply:
Ah. Well, adding this test won't help that really. You need to install rustfmt during the 'test' steps.